src/benchmarklib/file_based_query_generator.cpp

```diff
@@ -10,7 +10,8 @@
 
 namespace opossum {
 
-FileBasedQueryGenerator::FileBasedQueryGenerator(const BenchmarkConfig& config, const std::string& query_path) {
+FileBasedQueryGenerator::FileBasedQueryGenerator(const BenchmarkConfig& config, const std::string& query_path,
```

Comment:
Do we have a way to select which queries should be executed?

Reply:
We do not really, but I think we should have a way of only running a subset of the queries in the specified file/directory.

What do you think of two CLI options (names subject to improvement):

- `--query_path`, which specifies the file (hyriseBenchmarkFileBased only) or the directory (hyriseBenchmarkFileBased and hyriseBenchmarkJoinOrder).
- `--queries`, which is `all` by default but could also be a comma-separated list of queries (e.g. `--queries="1a,1b,1c,1d"`)?

Reply:
Sounds good. I'd also change `--tables` to `--table_path`